dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphHostControl.xaml.cs

```diff
@@ -42,16 +41,16 @@ namespace VisualLayerIntegration
     /// </summary>
     public partial class BarGraphHostControl : UserControl
     {
-        private CompositionHost compositionHost;
-        private Compositor compositor;
-        private Windows.UI.Composition.ContainerVisual graphContainer;
-        private BarGraph currentGraph;
+        private CompositionHost _compositionHost;
+        private Compositor _compositor;
+        private Windows.UI.Composition.ContainerVisual _graphContainer;
+        private BarGraph _currentGraph;
 
-        private double currentDpiX = 96.0;
-        private double currentDpiY = 96.0;
+        private double _currentDpiX = 96.0;
+        private double _currentDpiY = 96.0;
 
-        protected WindowRenderTarget windowRenderTarget;
-        private static RawColor4 white = new RawColor4(255, 255, 255, 255);
+        protected WindowRenderTarget _windowRenderTarget;
```

Comment:
Avoid protected fields. If it really needs to be protected make it a property with a private setter.

Reply:
updated